spec/models/notification_spec.rb

```diff
@@ -21,79 +21,55 @@
   end
 
   describe "when trying to create duplicate notifications" do
-    # Duplicate notifications are not allowed even when validations are skipped
-    it "doesn't allow to create a duplicate notification via import" do
```

Comment:
here we're testing activerecord-import, we're not testing the actual code, which at this moment in time wasn't even called at all :D 

replaced it with some safety tests